report-ng/app/src/components/log-view/virtual-log-view.ts

```diff
@@ -79,23 +79,31 @@ export class VirtualLogView extends AbstractLogView {
         this._calculateOccurrences();
     }
 
+    private checkMatches(logMessage: ILogEntry){
+        const foundInMessage = logMessage.message.match(this.searchRegexp);
+
+        const foundInStackTrace = logMessage.stackTrace
+            .flatMap(stackTrace => [
+                ...(stackTrace.stackTraceElements || []),
+                stackTrace.message])
+            .filter(line => line.match(this.searchRegexp));
+        const foundInLoggerName = this.statusConverter.separateNamespace(logMessage.loggerName).class.match(this.searchRegexp);
+
+        const logTime = new Date(logMessage.timestamp);
+        const formatTimeStamp = `{${logTime.getHours().toString().padStart(2,'0')}:${logTime.getMinutes().toString().padStart(2,'0')}:${logTime.getSeconds().toString().padStart(2,'0')}.${logTime.getMilliseconds().toString().padStart(3,'0')}}`
+        const foundInTimeStamp = formatTimeStamp.match(this.searchRegexp);
```

Comment:
Please use the value converter to get the same format and do not rebuild it manually.